src/User32.Desktop/User32+SpecialWindowHandles.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="SpecialWindowHandles"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Special window handles
+        /// </summary>
+        public enum SpecialWindowHandles
```

Comment:
Can they be used in place of normal handles ? If yes shouldn't they be constants with the same type as hwnd (`IntPtr` I believe)